### ISSUE TITLE
Strange Characters

### ISSUE THREAD

**Issue by comeback01** [Soulter/hugging-chat-api#230]
For some time now, regardless of the application I use, there are strange characters.

Thank you for your help.

![image](https://github.com/Soulter/hugging-chat-api/assets/17897116/5db834c8-f6b9-4c50-89a4-2cbd97d2b804)

**Comment by Whitelisted1** [comment]
I don't see this when printing to the console in Python, but with web tools I am able to see it

here is some of the output that might be causing this issue:
```json
{"type":"status","status":"started"}
{"type":"stream","token":"Hello\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"title","title":"👋 Hello"}
{"type":"stream","token":"!\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" It\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":"'s\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" nice\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" to\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" meet\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" you\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":".\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" Is\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" there\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" something\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" I\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" can\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" help\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" you\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" with\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":",\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" or\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" would\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" you\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" like\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" to\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":" chat\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"stream","token":"?\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"}
{"type":"finalAnswer","text":"Hello! It's nice to meet you. Is there something I can help you with, or would you like to chat?","interrupted":false}
```

I'm unsure why it would be outputting this, but a possible fix might be to replace the `\u0000` string with a blank string.

**Comment by Whitelisted1** [comment]
On further inspection, I am able to see the strange characters in the Python script
```
{'type': 'stream', 'token': 'Hello\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': '!\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' How\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' can\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' I\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' help\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' you\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': ' today\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
{'type': 'stream', 'token': '?\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'}
```

**Comment by comeback01** [comment]
Thank you very much for the explanation.

I didn't have this problem before, it's recent (about 10 days I think).

Is there a solution to this problem?

Thanks

**Comment by Soulter** [comment]
The problem has been solved, update to the latest version 0.4.8 to get the fixes.
